dojo/settings/settings.dist.py

```diff
@@ -273,12 +273,6 @@
     DD_RATE_LIMITER_ACCOUNT_LOCKOUT=(bool, False),
     # when enabled SonarQube API parser will download the security hotspots
     DD_SONARQUBE_API_PARSER_HOTSPOTS=(bool, True),
-    # when enabled, finding importing will occur asynchronously, default False
-    # This experimental feature has been deprecated as of DefectDojo 2.44.0 (March release). Please exercise caution if using this feature with an older version of DefectDojo, as results may be inconsistent.
-    DD_ASYNC_FINDING_IMPORT=(bool, False),
```

Comment:
If True, would it make sense to create an announcement banner or other alert to notify the users / admins that they are using a feature that is no longer present?

Reply:
I don't have the time right now to implement this.